Documentation/Devices-conventions.md

```diff
@@ -0,0 +1,25 @@
+# Conventions for devices APIs
+
+## Value returned by sensor
+
+- prefer property (i.e. Temperature) rather than method (i.e. GetTemperature)
```

Comment:
nit: start with upper case letter in every paragraph/line

Reply:
👍